app/assets/javascripts/discourse/components/scroll-tracker.js.es6

```diff
@@ -0,0 +1,42 @@
+import Scrolling from 'discourse/mixins/scrolling';
+
+export default Ember.Component.extend(Scrolling, {
+
+  didReceiveAttrs() {
+    this._super();
+
+    this.set('trackerName', 'scroll-tracker-' + this.get('name'))
+  },
+
+  didInsertElement() {
+    this._super();
+
+    this.bindScrolling({ name: this.get('name') });
+  },
+
+  didRender() {
+    this._super();
+
+    const data = this.session.get(this.get('trackerName'));
+    if (data && data.position && data.position >= 0 && data.tag == this.get('tag')) {
+      $(window).scrollTop(data.position + 1);
```

Comment:
also can you explain the `+ 1` please ? I just used it when I made it work, but not sure what's the reason behind this?